### ISSUE TITLE
Files remain greyed out after reconnecting CK2 project on a mounted FTP drive

### ISSUE THREAD

**Issue by Jaska** [bdkjones/CodeKit2#429]
I use Transmit FTP to mount external FTP drives. I then use Codekit2 to create projects there and process those files. This works surprisingly well, although it depends a lot on the server.

One thing that's annoyed me since CK2 was released is a problem when I open CK2, without having the required drives mounted. The project logically turns red, since the connection is broken. But if I then mount the drive from Transmit FTP and refresh the connection, the red color goes away - the connection appears fixed.

But all files remain grey. If I click on e.g a sass file, the menu doesn't come from the right side.

Refreshing the project or restarting CK2 doesn't work. Only thing that works is removing the project and adding it again.

OSX 10.9.4
Codekit 2.1.3
Transmit FTP 4.4.6

**Comment by mishamsk** [comment]
Same here with SSHFS (FUSE) to local VM on Parallels, CodeKit 2.1.7, OSX 10.9.5

**Comment by Jaska** [comment]
Just checked with 2.1.9 and the problem persists. Not that there has been any comments (e.g changelog) that it has been fixed.

**Comment by brendanvinson** [comment]
Experiencing this issue as well using a mounted network drive. If I lose connection to the server and remount the drive the files remain greyed out.

OSX 10.10.4
Codekit 2.10.3

**Comment by bdkjones** [comment]
Remove the project from the app and re-add it and you’ll be all set.

**Comment by brendanvinson** [comment]
That is what I ended up doing, but it's annoying to do that every time my vpn loses connectivity and drops the mounted drive (for instance, when my computer falls asleep).

Would be nice if on the first refresh after the invalid state it rediscovered the files.

**Comment by bdkjones** [comment]
Agreed. Unfortunately, I haven’t been able to determine why that happens across mount points. Basically, the alias ref CodeKit keeps track of invalidates when the mounted drive disappears. 

I hope to fix that in the next major release.

**Comment by randompixels** [comment]
The problem persists in CodeKit 2.8

**Comment by giagam** [comment]
hello @bdkjones please ignore my recent email. I just realised that this is the issue I am having. Hopefully CK3 will fix that. Bye.
